src/datastore/postgres/schema/v5.ts

```diff
@@ -0,0 +1,26 @@
+import { IDatabase } from "pg-promise";
+
+// tslint:disable-next-line: no-any
+export async function runSchema(db: IDatabase<any>) {
+    // Create schema
+    await db.none(`
+    CREATE TABLE metrics_users (
+        user_id TEXT NOT NULL,
+        remote BOOLEAN,
+        puppeted BOOLEAN
+    );
+
+    CREATE TYPE room_type AS ENUM ('user', 'channel');
+    CREATE TABLE metrics_rooms (
+        room_id TEXT NOT NULL,
+        type room_type
+    );
+
+    CREATE TABLE metrics_user_room_activities (
```

Comment:
Interesting to have a table of user+room activity, but I am wondering if it's strictly needed, and also if it's information we need to know.

Reply:
This table came out of the requirement to tell "active [rooms/users] for at least N days in a given month".
Each combination of (user, bridged room and date) will be a unique row, indicating one or more activities.